src/Microsoft.QualityGuidelines.Analyzers/Core/PreferJaggedArraysOverMultidimensional.cs

```diff
@@ -21,38 +24,131 @@ public abstract class PreferJaggedArraysOverMultidimensionalAnalyzer : Diagnosti
         private static readonly LocalizableString s_localizableMessageBody = new LocalizableResourceString(nameof(MicrosoftQualityGuidelinesAnalyzersResources.PreferJaggedArraysOverMultidimensionalMessageBody), MicrosoftQualityGuidelinesAnalyzersResources.ResourceManager, typeof(MicrosoftQualityGuidelinesAnalyzersResources));
         private static readonly LocalizableString s_localizableDescription = new LocalizableResourceString(nameof(MicrosoftQualityGuidelinesAnalyzersResources.PreferJaggedArraysOverMultidimensionalDescription), MicrosoftQualityGuidelinesAnalyzersResources.ResourceManager, typeof(MicrosoftQualityGuidelinesAnalyzersResources));
 
+        private const string helpLink = "https://msdn.microsoft.com/en-us/library/ms182277.aspx";
+
         internal static DiagnosticDescriptor DefaultRule = new DiagnosticDescriptor(RuleId,
```

Comment:
Why do we have multiple descriptors with the same rule ID? I see that you modified https://github.com/dotnet/roslyn-analyzers/issues/745 to also reflect that its ok to have multiple descriptors with same ID.